README.md

```diff
@@ -1,6 +1,50 @@
 [![Community Project header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
 
-# docs-website
+# New Relic Docs
+
+This repo contains the documentation website code and Markdown source files for  our preview Docs site: docs-preview.newrelic.com. When it's ready, all of our docs will live here.
+
+Read on to learn more about who we are and how you can contribute.
+
+## We'd like your help
+
+From early on, our Docs site has been managed by a team of writers. We've always welcomed contributions from anyone at New Relic. We credit our technical accuracy and comprehensive documentation to this openness. In a sense, it's documentation as conversation.
+
+Open sourcing our docs seemed like a great way to expand this conversation beyond New Relic.
```

Comment:
I like this "Open sourcing" sentence. Recommend moving it up to come right after "We've always welcomed contributions from anyone at New Relic." That way, we get the invitation in a little earlier. Though you will have to fool with the conversation part a little. Something like this:

From early on, our Docs site has been managed by a team of writers. We've always welcomed contributions from anyone at New Relic -- now we're open sourcing our docs to invite input from the broader community. We credit our technical accuracy and comprehensive documentation to this openness. In a sense, it's documentation as conversation. By making docs open source, we're expanding the conversation.

Reply:
Good suggestion!